addons/im_livechat/static/src/embed/common/chatbot/chatbot_service.js

```diff
@@ -149,6 +149,7 @@ export class ChatBotService {
             this.livechatService.thread?.messages.add({
                 ...rawMessage,
                 body: markup(rawMessage.body),
```

Comment:
Should use html: true

Reply:
There's no support of `html: true` with `RecordList.add()`. Only `Record.insert()` has it at the moment.